tests/Integrations/Symfony/V3_0/CommonScenariosTest.php

```diff
@@ -126,6 +126,37 @@ public function provideSpecs()
                             SpanAssertion::exists('symfony.kernel.terminate'),
                         ]),
                 ],
+                'A GET request to a missing route' => [
+                    SpanAssertion::build(
+                        'symfony.request',
+                        'symfony',
+                        'web',
+                        'GET /does_not_exist'
+                    )
+                        ->withExactTags([
+                            'http.method' => 'GET',
+                            'http.url' => 'http://localhost:9999/does_not_exist',
+                            'http.status_code' => '404',
+                        ])
+                        ->withChildren([
+                            SpanAssertion::exists('symfony.kernel.terminate'),
+                            SpanAssertion::exists('symfony.kernel.handle')->withChildren([
+                                SpanAssertion::exists('symfony.kernel.handleException')->withChildren([
+                                    SpanAssertion::exists('symfony.kernel.finish_request'),
+                                    SpanAssertion::exists('symfony.kernel.response'),
+                                    SpanAssertion::exists('symfony.kernel.exception')->withChildren([
+                                        SpanAssertion::exists('symfony.templating.render'),
+                                    ]),
+                                ]),
+                                SpanAssertion::exists('symfony.kernel.request')
+                                    ->setError(
```

Comment:
Doesn't this indicate that it _was_ an error? I thought the PR was to make 404's _not_ be an error?

Reply:
Correct - we still want to track the exception where it was thrown, but the root span should not be marked as an error. Before #995, the exception was getting attached to the root span which marked the whole trace as an error.